scalafmt-core/shared/src/main/scala/org/scalafmt/internal/FormatWriter.scala

```diff
@@ -411,6 +412,7 @@ class FormatWriter(formatOps: FormatOps) {
       // Insert a comma after b
       case TrailingCommas.always
           if !left.is[Comma] && !left.is[Comment] &&
+            !nextNonBlank(formatToken).right.is[Comment] &&
```

Comment:
Does this fail with `!formatToken.right.is[Comment]`?

Reply:
ah! That works, thanks. I was trying with `next(formatToken)` but that failed